### ISSUE TITLE
Crashes on REDMI 6 and ZTE Red Magic

### ISSUE THREAD

**Issue by hpk42** [deltachat/deltachat-android#1807]
From the gplay console we get crash reports with this stack trace (and some of the users left a 1-star review as a result) 

```
java.lang.NullPointerException: 
  at org.thoughtcrime.securesms.contacts.ContactAccessor.getAllSystemContactsAsString (ContactAccessor.java:74)
  at org.thoughtcrime.securesms.ContactSelectionListFragment$3.run (ContactSelectionListFragment.java:374)
```

![Screenshot from 2021-02-14 15-39-39](https://user-images.githubusercontent.com/73579/107879730-ddeaa380-6eda-11eb-9529-10492d0f659d.png)

**Comment by Hocuri** [comment]
@gerryfrancis suspected that Signal fixed it in https://github.com/signalapp/Signal-Android/commit/9838b2cf0ae24e01d41514e836151968922fc58f

**Comment by Hocuri** [comment]
Found the time to looks into this, but we already fixed this in 26c84975, we just didn't release yet :)

**Comment by hpk42** [comment]
great :)
